src/jdk.jdeps/share/classes/com/sun/tools/jdeps/resources/jdkinternals.properties

```diff
@@ -38,7 +38,7 @@ java.awt.peer=Should not use. See https://bugs.openjdk.org/browse/JDK-8037739
 java.awt.dnd.peer=Should not use. See https://bugs.openjdk.org/browse/JDK-8037739
 jdk.internal.ref.Cleaner=Use java.lang.ref.PhantomReference @since 1.2 or java.lang.ref.Cleaner @since 9
 sun.awt.CausedFocusEvent=Use java.awt.event.FocusEvent::getCause @since 9
-sun.font.FontUtilities=See java.awt.Font.textRequiresLayout	@since 9
```

Comment:
This tab just looked out of place, so I replaced it with a space. (Rhyming not intentional...)